runtime/service.go

```diff
@@ -320,10 +366,7 @@ func (s *service) Connect(ctx context.Context, req *taskAPI.ConnectRequest) (*ta
 func (s *service) Shutdown(ctx context.Context, req *taskAPI.ShutdownRequest) (*ptypes.Empty, error) {
 	log.G(ctx).WithFields(logrus.Fields{"id": req.ID, "now": req.Now}).Debug("shutdown")
 
-	if _, err := s.agentClient.Shutdown(ctx, req); err != nil {
-		log.G(ctx).WithError(err).Error("failed to shutdown agent")
-	}
-
+	s.agentClient.Shutdown(ctx, req)
```

Comment:
copied from closed pr [comment](https://github.com/firecracker-microvm/firecracker-containerd/pull/12#discussion_r236523314) from @mxpv: 
>Just curious: is it by mistake or logging was removed for some reason?

Reply:
Shutdown was not returning successfully. When looking at why it wasbecause the runc impl always returns an error because it just calls os.Exit() prior to returning [here](https://github.com/containerd/containerd/blob/master/runtime/v2/runc/service.go#L585).

Looking at it now though I think it makes more sense to ignore that result in the agent since the agent is doing more than just that runc call during shutdown. Will update it to have this logging back and ignore the always-error from calling runc.Shutdown inside the agent instead.